api/client/search.go

```diff
@@ -21,12 +23,25 @@ import (
 func (cli *DockerCli) CmdSearch(args ...string) error {
 	cmd := Cli.Subcmd("search", []string{"TERM"}, Cli.DockerCommands["search"].Description, true)
 	noTrunc := cmd.Bool([]string{"-no-trunc"}, false, "Don't truncate output")
-	automated := cmd.Bool([]string{"-automated"}, false, "Only show automated builds")
-	stars := cmd.Uint([]string{"s", "-stars"}, 0, "Only displays with at least x stars")
+	filter := cmd.String([]string{"f", "-filter"}, "", "Use filters (is-automated, is-official, has-stars)")
+
+	// Deprecated since Docker 1.13 in favor of "--filter"
+	automated := cmd.Bool([]string{"-automated"}, false, "Only show automated builds - DEPRECATED")
+	stars := cmd.Uint([]string{"s", "-stars"}, 0, "Only displays with at least x stars - DEPRECATED")
```

Comment:
can you put a `#` before the flag names? that will automatically output a deprecation message if someone is using them